gym/utils/play.py

```diff
@@ -97,11 +99,12 @@ def display_arr(
 def play(
     env: Env,
     transpose: Optional[bool] = True,
-    fps: Optional[int] = 30,
+    fps: Optional[int] = None,
     zoom: Optional[float] = None,
     callback: Optional[Callable] = None,
-    keys_to_action: Optional[Dict[Tuple[int], int]] = None,
+    keys_to_action: Optional[Dict[Union[Tuple[int], str], ActType]] = None,
```

Comment:
I realised that a `Tuple[str]` is unnecessary but it feels weird compared to passing a `Tuple[int]` and took me a while to work out why we don't need it. Could you add a comment to the docstring on this